Content.Server/Audio/Jukebox/JukeboxSystem.cs

```diff
@@ -75,6 +77,16 @@ private void OnJukeboxSetTime(EntityUid uid, JukeboxComponent component, Jukebox
         }
     }
 
+    //SS220-jukebox-tweak-begin
+    private void OnJukeboxSetGain(EntityUid uid, JukeboxComponent component, JukeboxSetGainMessage args)
+    {
+        Audio.SetGain(component.AudioStream, args.Gain);
+
+        component.Gain = args.Gain;
+        Dirty(uid, component);
+    }
```

Comment:
Совсем забыл, нужна валидация на стороне сервера что злоумылшенник не передал быдло-данные, пишу по памяти, проверить что после этого настройка звука всё ещё работает!

```suggestion
    {
        var gain = Math.Clamp(args.Gain, 0f, 1f);
        Audio.SetGain(component.AudioStream, gain);
        component.Gain = gain;
        Dirty(uid, component);
    }
```